website/src/content/docs/docs/handbook/breaking-change-policy.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Breaking Change Policy
+---
+
+# Breaking Change Policy
+
+import { Badge } from "@astrojs/starlight/components";
+
+:::note
+This is to be applicable after 1.0.0 release
+:::
+
+## Versioning Scheme
+
+TypeSpec Compiler and libraries follow [semver](https://semver.org/) for versioning. Which means a breaking change(according the following policy) will only be introduced in a major version release.
+
+## Breaking Change Philosophy
+
+The basic of the policy can be summarized as follows, we guarantee stability for runtime behavior but not for the build process.
```

Comment:
```suggestion
The basic of the policy can be summarized as follows: existing language syntax and semantics will not change without a major version. The runtime behavior for a given TypeSpec will not change without a major version. TypeSpecs leveraging new or updated features may cause runtime API consumers to fail (e.g. because a new type is unhandled) or builds to fail (e.g. because exhaustive unions are no longer exhaustive).
``